pkg/stores/partition/listprocessor/processor.go

```diff
@@ -260,17 +260,15 @@ func getLimit(apiOp *types.APIRequest) int {
 
 // FilterList accepts a channel of unstructured objects and a slice of filters and returns the filtered list.
```

Comment:
This is no longer a channel
```suggestion
// FilterList accepts a slice of unstructured objects and a slice of filters and returns the filtered list.
```